numpy/core/tests/test_dtype.py

```diff
@@ -252,6 +252,9 @@ def test_from_dictproxy(self):
         dt2 = np.dtype((np.void, dt.fields))
         assert_equal(dt2.fields, dt.fields)
 
+    def test_bool_commastring(self):
+        np.dtype('?,?,?') #raises?
```

Comment:
I would prefer if you actually assert some things that are correct on the result dtype, so we know it works correctly as well ;).